examples/PowerElectronics/DistributedGeneratorTest/DGTest.cpp

```diff
@@ -89,5 +89,7 @@ int main(int /* argc */, char const** /* argv */)
   }
   std::cout << "Test successful!\n";
 
+  delete dg;
```

Comment:
I personally would prefer a smart pointer here. If we ever push linting through, this will be a lint warning.

Reply:
Can it just be created on the stack?
```
GridKit::DistributedGenerator<double, size_t> dg(0, parms, true);
```

Reply:
Yes, it can. For the examples, that would be my preferred way to go about it.

Reply:
i've moved it to the stack